ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -172,11 +172,17 @@
   (days-to-first-diff start-date date->hash))
 
 (defn- init-view-state [app scope]
-  (let [initial-view-state {:all-route-changes-display? false
+  (let [current-year (.getFullYear (js/Date.))
+        initial-view-state {:all-route-changes-display? false
                             :all-route-changes-chenckbox false
                             :open-sections {:gtfs-package-info false}
                             :scope scope
-                            :service-changes-for-date-loading? true}]
+                            :service-changes-for-date-loading? true
+                            :show-next-year? (or
+                                               (t/after?
+                                                 (goog.date.DateTime. (js/Date.))
+                                                 (goog.date.DateTime. (js/Date. current-year 8 1)))
```

Comment:
Maybe add a somekind comment why Aug 1st was hard-coded (out of convenience when to display the following year?)